crates/ruff/src/rules/pycodestyle/rules/logical_lines/missing_whitespace.rs

```diff
@@ -63,7 +63,11 @@ pub(crate) fn missing_whitespace(
             TokenKind::Comma | TokenKind::Semi | TokenKind::Colon => {
                 let after = line.text_after(token);
 
-                if !after.chars().next().map_or(false, char::is_whitespace) {
+                if !after
+                    .chars()
+                    .next()
+                    .map_or(false, |c| char::is_whitespace(c) || c == '\\')
```

Comment:
Empirically, pycodestyle seems to allow continuations:

```py
#: Okay
a = (1,\
2)
```

We could deviate and remove this change, I don't feel strongly, but it was necessary to achieve compatibility.